README.md

```diff
@@ -14,57 +14,73 @@ Building and running this node allows you to:
 For full documentation and a project overview, go to
 [Livepeer Documentation](https://github.com/livepeer/wiki/wiki)
 
-## Installation
-The easiest way to install Livepeer is by downloading it from the [release page on Github](https://github.com/livepeer/go-livepeer/releases).  Pick the appropriate platform and the latest version.
+## Installing Livepeer
+### Option 1: Download executables
+The easiest way to install Livepeer is by downloading the `livepeer` and `livepeer_cli` executables from the [release page on Github](https://github.com/livepeer/go-livepeer/releases). 
 
-## Build
-If you have never set up your Go programming environment, do so according to Go's [Getting Started Guide](https://golang.org/doc/install).
+1. Download the packages for your OS - darwin for Macs and linux for linux. 
+2. Rename them to `livepeer` and `livepeer_cli`
+3. Make sure they have executable permissions by running `chmod +x livepeer` and `chmod +x livepeer_cli`
```

Comment:
Note to self that we should redo the process for generating these binaries to just have one linux and one windows .tar.gz that contains the already permissioned livepeer and livepeer_cli files. Then we can get rid of the renaming and the chmod'ing.

Reply:
tracked here: https://github.com/livepeer/go-livepeer/issues/131